Source/WebCore/Modules/mediastream/MediaStreamTrack.h

```diff
@@ -190,7 +190,7 @@ class MediaStreamTrack
 
     ScriptExecutionContext* scriptExecutionContext() const final { return ActiveDOMObject::scriptExecutionContext(); }
         
-    Ref<MediaStreamTrackPrivate> m_private;
+    const Ref<MediaStreamTrackPrivate> m_private;
```

Comment:
Oh, you've actually marked it as const already. So your changes above are not needed at all.